include/lsst/jointcal/Associations.h

```diff
@@ -108,6 +109,9 @@ class Associations {
 
     size_t getMaxMeasuredStars() const { return _maxMeasuredStars; }
 
+    double getEpoch() const { return _epoch; }
+    void setEpoch(double epoch) { _epoch = epoch; }
```

Comment:
While this is a trivial method, copy the field's comment here for doxygen docstring.

"julian epoch year (e.g. 2000.0 for J2000)"